### ISSUE TITLE
The training logs of Model 2 (DDPG) and Model 4 (TD3) are weird

### ISSUE THREAD

**Issue by JinChengneng** [AI4Finance-LLC/FinRL-Library#33]
While running ```FinRL_single_stock_trading.ipynb```, I found that the logs of Model 2 (DDPG) and Model 4 (TD3) are weird. It looks like none trading has been done, which is not same as the demo you give.  I would appreciate it if you could help to check if there is anything wrong.
BTW, there is a little typo that SAC model should be marked as Model 5 rather than Model 4.

```
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 4        |
|    fps             | 75       |
|    time_elapsed    | 133      |
|    total timesteps | 10064    |
| train/             |          |
|    actor_loss      | -513     |
|    critic_loss     | 496      |
|    learning_rate   | 0.0001   |
|    n_updates       | 7548     |
---------------------------------
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 8        |
|    fps             | 64       |
|    time_elapsed    | 310      |
|    total timesteps | 20128    |
| train/             |          |
|    actor_loss      | -330     |
|    critic_loss     | 432      |
|    learning_rate   | 0.0001   |
|    n_updates       | 17612    |
---------------------------------
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 12       |
|    fps             | 60       |
|    time_elapsed    | 495      |
|    total timesteps | 30192    |
| train/             |          |
|    actor_loss      | -172     |
|    critic_loss     | 194      |
|    learning_rate   | 0.0001   |
|    n_updates       | 27676    |
---------------------------------
```

```
Logging to tensorboard_log/td3/td3_1
begin_total_asset:100000
end_total_asset:687696.4228235682
total_reward:587696.4228235682
total_cost:  10571.455648996345
total_trades:  2513
Sharpe:  0.944539208399143
=================================
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 4        |
|    fps             | 60       |
|    time_elapsed    | 165      |
|    total timesteps | 10064    |
| train/             |          |
|    actor_loss      | 3.17e+03 |
|    critic_loss     | 1.28e+04 |
|    learning_rate   | 0.0003   |
|    n_updates       | 7548     |
---------------------------------
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 8        |
|    fps             | 51       |
|    time_elapsed    | 392      |
|    total timesteps | 20128    |
| train/             |          |
|    actor_loss      | 2.96e+03 |
|    critic_loss     | 7.85e+03 |
|    learning_rate   | 0.0003   |
|    n_updates       | 17612    |
---------------------------------
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
begin_total_asset:100000
end_total_asset:100000.0
total_reward:0.0
total_cost:  0
total_trades:  0
---------------------------------
| time/              |          |
|    episodes        | 12       |
|    fps             | 48       |
|    time_elapsed    | 625      |
|    total timesteps | 30192    |
| train/             |          |
|    actor_loss      | 2.4e+03  |
|    critic_loss     | 5.31e+03 |
|    learning_rate   | 0.0003   |
|    n_updates       | 27676    |
---------------------------------
```

**Comment by BruceYanghy** [comment]
Lower the learning rate: divided by 3, 5, and 10, also lower the timesteps, and try again. I also experienced this problem when learning rate is high, model just stuck.

**Comment by JinChengneng** [comment]
Got it! Thanks a lot.

**Comment by JinChengneng** [comment]
One more question about the logs. Could you please give an explanation of the meaning of the variables during training, such as ```approx_kl```, ```clip_fraction```, ```entropy_loss```, ```std``` and so on? Moreover, which loss should we pay more attention to while training? Thanks a lot.
